### PR TITLE
Update resource model admin grid tut

### PR DESCRIPTION
**Pull request by gixid192**
## Purpose of this pull request

Add some classes so that readers will not need to refer to the git.

Details: When copy/paste from the devdocs, I get some error, this PR fixes the error I encountered
1. Cannot find `HttpGetActionInterface` class
2. Resource Model is defined in the di.xml file but its class code is not in the devdocs (the git however has it)

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/extension-dev-guide/admin-grid.html

### CODE REVIEW

File: src/guides/v2.3/extension-dev-guide/admin-grid.md [hunk 1]
```diff
@@ -377,6 +377,15 @@ class Collection extends SearchResult
 
 It uses a custom collection file to add custom filters to map, and makes the grid filters work with the ID and name fields. Without `addFilterToMap`, you will not be able to search within the `name` column.
 
+The resource model class translates into `app/code/Dev/Grid/Model/ResourceModel/Category.php`:
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/admin-grid.md:380]:
```suggestion
The resource model class translates to `app/code/Dev/Grid/Model/ResourceModel/Category.php`:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/extension-dev-guide/admin-grid.md (hunk 1 above)
Comment (gixid192) [reply src/guides/v2.3/extension-dev-guide/admin-grid.md:380]: This line is copied, not mine :D Right above is the original sentence, which is `The collection class translates into app/code/Dev/Grid/Ui/DataProvider/Category/Listing/Collection.php`

Do you want me to change it as well?

**Review by dobooth — APPROVED** [review]

File: src/guides/v2.3/extension-dev-guide/admin-grid.md (hunk 1 above)
Comment (BarnyShergold) [reply src/guides/v2.3/extension-dev-guide/admin-grid.md]: yes please - "translates to" is better English than "translates into"

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8851`.

**Pull request merged**